src/components/Favorite/Favorite.js

```diff
@@ -1,12 +1,30 @@
 import styles from './Favorite.module.scss';
 import PageTitle from '../PageTitle/PageTitle';
+import { useSelector } from 'react-redux';
+import {getAllFavorites} from '../../redux/store';
+import Card from '../Card/Card';
 
 const Favorite = () => {
+  
+  const cards = useSelector(state => getAllFavorites(state));
+  
+  if (cards.length === 0) {
+    return (
+      <>
+        <PageTitle>Favorite</PageTitle>
+        <article className={styles.column}>
+          <p>You don't have favorite cards!</p>
```

Comment:
Różnica pomiędzy tymi dwoma renderami to tak naprawdę linia 16 i 25. Zamiast tego użyj operatora trojargumentowego bezpośredino w jsxie:

```
{ cards.length === 0 ? ( <p>...</p>) : cards.map() }
````

Reply:
Fixed